src/Installers/Windows/AspNetCoreModule-Setup/ANCMVS.wixproj

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <OutputNamePrefix>aspnetcoremodulevs-</OutputNamePrefix>
+    <OutputNameSuffix>-win-$(Platform)</OutputNameSuffix>
+    <ProductNameFolder>Microsoft ASP.NET Core Module</ProductNameFolder>
+    <ProductNameShort>AspNetCoreModule.VS</ProductNameShort>
+    <OutputType>Package</OutputType>
+    <ProjectGuid>541E1EF7-E129-4274-A05A-321F785AA2E9</ProjectGuid>
+    <IsShipping>true</IsShipping>
+    <DefineConstants>$(DefineConstants);ProductNameFolder=$(ProductNameFolder)</DefineConstants>
+    <DefineConstants>$(DefineConstants);ProductNameShort=$(ProductNameShort)</DefineConstants>
+    <NamespaceGuid>$(ProjectGuid)</NamespaceGuid>
```

Comment:
@joeloff do I need  to do anything specific with these GUIDs?

Reply:
no, the namespace guid would only be used to generate things like the ProjectCode/UpgradeCode (assuming this project uses the same task the runtime/targeting packs use)

Reply:
I think they use the same tasks, I hit an error when I didn't have a guid.